src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -124,8 +141,33 @@ class ActiveFocusInvestigation extends React.Component<
       .then((result: Plan[]) => fetchPlansActionCreator(result))
       .catch(err => displayError(err));
   }
-  public handleSubmit(event: React.FormEvent<HTMLFormElement>) {
-    event.preventDefault();
+
+  public debouncedSearch(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
How / where is this being debounced? Looks like it fires every time the input value is updated (which should be fine I think) 🤔